src/appleseed/renderer/modeling/camera/camera.cpp

```diff
@@ -295,6 +300,44 @@ double Camera::extract_near_z() const
     return near_z;
 }
 
+// Assumed shutter opening/closing curve is straight line.
+float Camera::integrate_inverse_curve(float a, float b, float x) const
```

Comment:
This method isn't really specific to the `Camera` class, it makes sense to transform it into a local free function:

    namespace
    {
        float integrate_inverse_curve(float a, float b, float x) const
        {
            ...
        }
    }

Other remarks:
- We need to find a better name, in particular the name must reflect that it's a linear curve. Maybe `map_sample_to_linear_curve()`?
- make `a`, `b` and `x` const